File: 250830/5의 배수의 개수는?/the-number-of-multiples-of-5.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[][] num = new int[4][4];
        int total = 0;

        for(int i=0;i<4;i++) {
            for(int j=0;j<4;j++) {
                num[i][j] = sc.nextInt();
                if(num[i][j]%5==0) {
                total++;
                }
            }
        }

        System.out.print(total);

    }
}

File: 250830/5의 배수의 개수는?/the-number-of-multiples-of-5.c
#include <stdio.h>

int main() {
    int num[4][4];
    int total = 0;

    for(int i=0;i<4;i++) {
        for(int j=0;j<4;j++) {
            scanf("%d ", &num[i][j]);
            if(num[i][j]%5==0) {
                total++;
            }
        }
    }

    printf("%d", total);

    return 0;
}